x/inflation/spec/01_concepts.md

```diff
@@ -52,20 +52,17 @@ The inflation distribution for staking, usage incentives and community pool is
 implemented through an exponential formula, a.k.a. the Half Life.
 
 Inflation is minted in daily epochs. During a period of 365 epochs (one year), a
-daily provision of Evmos tokens is minted and allocated to staking rewards,
-usage incentives and the community pool (`epochProvison`). The epoch provision
-doesn’t change within a period and its cumulated amount per period is equal
-total inflation per period (`epochProvision * epochsPerPeriod =
-periodProvision`).
+daily provision (`epochProvison`) of Evmos tokens is minted and allocated to staking rewards,
+usage incentives and the community pool. The epoch provision depends on parameters and is calculated every day.
 
-At the end of each period , the epoch provision is recalculated with the
-calculation below:
+At the end of each epoch, the epoch provision is recalculated with the calculation below:
```

Comment:
The last sentence before this is already saying that it's recalculated every day and the paragraph says that inflation is using daily epochs - IMO it's repeated too often if it's also mentioned here again
```suggestion
The calculation of the epoch provision is done according to the following formula:
```